python/tank/util/qt_importer.py

```diff
@@ -111,7 +125,7 @@ def _import_pyside(self):
 
         :returns: The (binding name, binding version, modules) tuple.
         """
-        from PySide import QtCore, QtGui
+        from PySide import QtCore, QtGui, QtWebKit, QtNetwork
```

Comment:
I searched the Qt/PyQt/PySide to see if these modules were optional or always present... I was under the impression that if you could find QtCore, QtWebKit and QtNetwork would successfully load... But I may be wrong. I will wrap the import in a `try/except`